data/saifrahmn.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://github.com/saifrahmn.png",
+    "links": [
+      {
+        "name": "For Collaboration",
+        "url": "https://github.com/saifrahmn",
+        "icon": "github"
```

Comment:
these are not validate icons, please read the docs https://linkfree.eddiehub.io/docs/quickstart